spec/factories/reservation.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :reservation do
+    association :screening
```

Comment:
associations may cause problems when the app grows

Nothing wrong here just something to keep in mind